app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+require './lib/user_helper.rb'
+
+class User < ApplicationRecord
+  include UserHelper
+
+  has_many :posts
+  has_many :comments
+  validates :user_name, uniqueness: true, length: { in: 6..15 }
```

Comment:
- [ ] Add [presence validation](https://guides.rubyonrails.org/active_record_validations.html#presence) for the user_name